app/containers/Avatar/interfaces.ts

```diff
@@ -1,23 +1,23 @@
 export interface IAvatar {
```

Comment:
We can not remove a `Partial` and set all components as `optional`... 
Take a look at wich components is required or not, please.

Reply:
Yeap, I did, but I didn't figure out any param that must be required. WDYT in this case?